test/clinic.test.js

```diff
@@ -1895,7 +1879,6 @@ describe("app", function() {
                             assert.deepEqual(metrics['test.clinic.avg.sessions_to_register'].values, [5]);
                             assert.deepEqual(metrics['test.clinic.percent_incomplete_registrations'].values, [25]);
                             assert.deepEqual(metrics['test.clinic.percent_complete_registrations'].values, [75]);
-                            assert.deepEqual(metrics['test.sum.subscriptions'].values, [1]);
```

Comment:
Please make an issue for ndoh-control which captures metrics that were removed from go-ndoh that need adding to python so we don't lose any inadvertently